Userland/Services/WebDriver/Client.h

```diff
@@ -49,6 +49,7 @@ class Client final : public Web::WebDriver::Client {
     virtual Web::WebDriver::Response refresh(Web::WebDriver::Parameters parameters, JsonValue payload) override;
     virtual Web::WebDriver::Response get_title(Web::WebDriver::Parameters parameters, JsonValue payload) override;
     virtual Web::WebDriver::Response get_window_handle(Web::WebDriver::Parameters parameters, JsonValue payload) override;
+    virtual Web::WebDriver::Response new_window(Web::WebDriver::Parameters parameters, JsonValue payload) override;
```

Comment:
Same re: spec order (though on this side of the IPC connection, belongs below get_window_handles)